src/libraries/System.Net.Requests/tests/HttpWebRequestTest.cs

```diff
@@ -2138,13 +2235,15 @@ public async Task SendHttpPostRequest_WithContinueTimeoutAndBody_Success(bool ex
         [InlineData(false)]
         public async Task SendHttpPostRequest_When100ContinueSet_ReceivedByServer(bool expect100Continue)
         {
+            TaskCompletionSource tcs = new TaskCompletionSource();
```

Comment:
Why is this change needed?

Reply:
I believe this is not needed, I'll remove it.